### PR TITLE
[DESK] Improve pt-PT Translation; Add missing accel keys and fix some others accel collisions.

### PR DESCRIPTION
**Pull request by zecarlos1957**
Add file header. 
Change SUBLANG to NEUTRAL
Improve Portuguese (pt-PT) translation
Add missing accel keys and fix some others accel collisions.

### CODE REVIEW

File: dll/cpl/desk/lang/pt-PT.rc
Comment (binarymaster) [inline dll/cpl/desk/lang/pt-PT.rc:2]:
```suggestion
 * PROJECT:     ReactOS Display Control Panel
```

**Review by HBelusca — APPROVED** [review]

File: dll/cpl/desk/lang/pt-PT.rc
Comment (JoachimHenze) [inline dll/cpl/desk/lang/pt-PT.rc:45]: Yes, better than before with the collision. But since I notidced recently that &t doesn't look very good (very narrow character) I do wonder whether MS uses &t here as well, or something else. Can you prove with a pt-PT MS-pic?

File: dll/cpl/desk/lang/pt-PT.rc
Comment (JoachimHenze) [inline dll/cpl/desk/lang/pt-PT.rc:189]: Why is that line commented out? Is that also like that in other languages? Should the line be stripped or the comment be removed?

**Review by JoachimHenze — APPROVED** [review]
Looks ok to me in  general. Let's wait for an answer to the &t question before the merge.

File: dll/cpl/desk/lang/pt-PT.rc
Comment (zecarlos1957) [reply dll/cpl/desk/lang/pt-PT.rc:45]: VirtualBox_WinSrv2003_Monitor
There he is.

Comment (JoachimHenze) [reply dll/cpl/desk/lang/pt-PT.rc:45]: MS is king. Approved then. Thx.

File: dll/cpl/desk/lang/pt-PT.rc
Comment (zecarlos1957) [reply dll/cpl/desk/lang/pt-PT.rc:189]: I don't know, it wasn't me.
This text exists in win2003 Srv, and is of great help to the user.
Can I reset visible text?

**Pull request merged**
